src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/target.py

```diff
@@ -48,13 +48,13 @@ def _define_target_type(target):
         return target
 
     def generate_waves(self):
-        waves = []
         wave_size = self.wave.get('size', 50)
+        wave = []
         length = len(self.account_list)
         for index in range(0, length, wave_size):
-            yield self.account_list[index:min(index + wave_size, length)]
-            waves.append(self.account_list[index:min(index + wave_size, length)])
-        return waves
+            wave.append(self.account_list[index:min(index + wave_size, length)])
```

Comment:
I've changed this back as it had issues serializing the response due to a generator being used.